radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -749,6 +765,10 @@ public Result transferTokens(
 		return this.execute(transferTokensAction);
 	}
 
+	private long generateTimestamp() {
+		return System.currentTimeMillis();
+	}
+
 	/**
 	 * Returns an unsigned atom with the appropriate fees given a list of
```

Comment:
update documentation of method, says it returns `"...an unsigned atom..."`, but we are signing it :)

Reply:
Hmm? the atom returned in the method is still unsigned?